code/modules/mob/new_player/preferences_setup.dm

```diff
@@ -1,10 +1,10 @@
 /datum/preferences
 	//The mob should have a gender you want before running this proc. Will run fine without H
 /datum/preferences/proc/random_character(gender_override)
-	var/datum/species/S = new all_species[species]
+	var/datum/species/S = all_species[species]

```

Comment:
Totally right, this species var being a unique type isn't necessary for this proc. Silly me